test/client/components/account.test.js

```diff
@@ -3,10 +3,11 @@ import { mount, shallow } from 'enzyme';
 import { MemoryRouter } from 'react-router-dom';
 import renderer from 'react-test-renderer';
 import { AccountPage } from '../../../public/src/components/account/account';
-import { fetchUser } from '../../../public/src/actions/account_info';
-import deleteUser from '../../../public/src/actions/edit_account';
+import { fetchUser } from '../../../public/src/actions/users';
 
-jest.mock('../../../public/src/actions/edit_account', () => jest.fn());
+// jest.mock('../../../public/src/actions/edit_account', () => jest.fn());
```

Comment:
Are we satisfied with the mocking below or do we need to keep this commented older mock around?

Reply:
Deleted it.